src/app/games/game-summaries/external-game/external-game.component.html

```diff
@@ -0,0 +1,26 @@
+<mat-card>
+  <a [href]="data.routingURL">
```

Comment:
I noticed that `game-summary` is still concatenating the href as opposed to using routingURL here. Could this and any other inconsistencies be matched up?

Reply:
But that's an internal link, so doesn't it work differently anyway?

Reply:
Is the idea to add `"routingURL": "games/oot"` or similar to all of them, add that to the game component, and use that?

Reply:
I think I did this correctly in the latest commit.